strings/check_isogram.py

```diff
@@ -0,0 +1,28 @@
+"""
+wiki: https://en.wikipedia.org/wiki/Heterogram_(literature)#Isograms
+"""
+
+
+def check_isogram(string: str) -> bool:
```

Comment:
This is a slightly more self-documenting name for a function that returns a bool.
```suggestion
def is_isogram(string: str) -> bool:
```

Also, please add a test for `true123`.  Should that return True or raise a ValueError?

Reply:
Renamed function and added test for when a string contains 1 or more digits. The latter now raises a ValueError. @cclauss